src/cocoa/BugsnagCocos2dxPlugin.h

```diff
@@ -1,13 +1,52 @@
-#import <Foundation/Foundation.h>
+#import <Bugsnag/Bugsnag.h>
 
 @interface BugsnagCocos2dxPlugin : NSObject
 
 /**
- * Register the Bugsnag Cocos2d-x plugin, prior to calling [Bugsnag start]
+ * Start listening for crashes.
  *
- * @param cocos2dVersion The version cocos2d being used, available via
- *                       cocos2d::cocos2dVersion()
+ * This method initializes Bugsnag with the configuration set in your Info.plist.
+ *
+ * If a Bugsnag apiKey string has not been added to your Info.plist or is empty, an
+ * NSException will be thrown to indicate that the configuration is not valid.
+ *
+ * Once successfully initialized, NSExceptions, C++ exceptions, Mach exceptions and
+ * signals will be logged to disk before your app crashes. The next time your app
+ * launches, these reports will be sent to your Bugsnag dashboard.
+ */
++ (BugsnagClient *_Nonnull)start;
```

Comment:
In Objective-C method declarations it's preferable to use the nicer syntax available (not available for C function or block declarations):

```suggestion
+ (nonnull BugsnagClient *)start;
```